.travis/script.sh

```diff
@@ -484,7 +484,8 @@ set -x
     git clean -xfd || die
     ./bootstrap || die
     ./configure                             \
-        --enable-ncp-app=all                \
+        --enable-ncp                        \
+        --enable-ftd                        \
```

Comment:
This was previously `all`. Is the omission of `mtd` intentional here?

Reply:
no. thanks for noticing.